docs/sdk/server-side-sdks/dotnet/dotnet.md

```diff
@@ -0,0 +1,35 @@
+---
+title: .NET SDK
+---
+
+import CustomDocCardList from '@site/src/components/CustomDocCardList'
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
+
+# DevCycle .NET / C# SDK
+
+The DevCycle .NET / C# SDK has two operation modes; Cloud Bucketing - which interfaces with the [DevCycle Bucketing API](/bucketing-api/#tag/devcycle), 
+and Local Bucketing - which performs bucketing locally.
+
+## Operational Differences between Cloud and Local Bucketing
```

Comment:
You might be able to get away with just referencing this section of the docs instead of reiterating it here.. https://docs.devcycle.com/sdk/#difference-between-local-and-cloud-bucketing